Update autoexpanding-archiving.md

jcreigt opened the pull request:
Clarified OWA and Outlook Win32 support for hybrid exchange deployments (primary account on-premises and archive in cloud). 
Also updated Win32 auto-expanding search availability to MEC channel.

jcreigt committed:
Update autoexpanding-archiving.md

Clarified OWA and Outlook Win32 support for hybrid exchange deployments (primary account on-premises and archive in cloud). 
Also updated Win32 auto-expanding search availability to MEC channel.

cabailey commented:
@jcreigt Thanks for creating this PR but please address comments in email before I merge it

JohanFreelancer9 commented on microsoft-365/compliance/autoexpanding-archiving.md:
```diff
@@ -89,9 +89,9 @@ Here are some things to consider when using Outlook or Outlook on the web to acc
 - You can delete items in an auto-expanded storage area. However, you can't use the Recover Deleted Items feature to recover an item after auto-expanding archiving is enabled for a mailbox.
 
-- Search for auto-expanded archiving is available in Outlook for the web (OWA). Similar to Online Archive, you can search for items that were moved to an additional storage area. When archive is selected as the search scope in OWA, all archives (including auto-expanded archives) and their corresponding subfolders will be searched.
+- Search for auto-expanded archiving is available in Outlook for the web (OWA). Similar to Online Archive, you can search for items that were moved to an additional storage area. When archive is selected as the search scope in OWA, all archives (including auto-expanded archives) and their corresponding subfolders will be searched. Note that search is not supported for auto-expanding archive featu ...
```
```suggestion
- Search for auto-expanded archiving is available in Outlook for the web (OWA). Similar to Online Archive, you can search for items that were moved to an additional storage area. When archive is selected as the search scope in OWA, all archives (including auto-expanded archives) and their corresponding subfolders will be searched. Note that search is not supported for the auto-expanded archive feature in a cloud-only archive situation (primary mailbox still on-premises).
```

JohanFreelancer9 commented on microsoft-365/compliance/autoexpanding-archiving.md:
```diff
@@ -89,9 +89,9 @@ Here are some things to consider when using Outlook or Outlook on the web to acc
+- Search for auto-expanded archiving is available in Outlook for the web (OWA). Similar to Online Archive, you can search for items that were moved to an additional storage area. When archive is selected as the search scope in OWA, all archives (including auto-expanded archives) and their corresponding subfolders will be searched. Note that search is not supported for auto-expanding archive featu ...
 
-- Auto-expanded archive search is available in Outlook Desktop in Current Channel (Preview). Within this preview, the Current Mailbox scope is available, thus allowing you to search the auto-expanded archive. For more information about this and other Microsoft Search support features, see [How Outlook for Windows connected to Exchange Online utilizes Microsoft Search](https://techcommunity.micros ...
+- Auto-expanded archive search is available in Outlook Desktop in Monthly Enterprise Channel. Within this preview, the Current Mailbox scope is available, thus allowing you to search the auto-expanded archive. Note that search is not supported for auto-expanding archive feature in a cloud-only archive situation (primary mailbox still on-premises). For more information about this and other Microso ...
```
```suggestion
- Auto-expanded archive search is available in Outlook Desktop in Monthly Enterprise Channel. Within this preview, the Current Mailbox scope is available, thus allowing you to search the auto-expanded archive. Note that search is not supported for the auto-expanded archive feature in a cloud-only archive situation (primary mailbox still on-premises). For more information about this and other Microsoft Search support features, see [How Outlook for Windows connected to Exchange Online utilizes Microsoft Search](https://techcommunity.microsoft.com/t5/outlook-global-customer-service/how-outlook-for-windows-connected-to-exchange-online-utilizes/ba-p/1715045). 
```

JohanFreelancer9 reviewed (changes requested):
Copy review - @jcreigt, kindly commit the minor suggested changes. Many thanks.

jcreigt committed:
Update microsoft-365/compliance/autoexpanding-archiving.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

jcreigt committed:
Update microsoft-365/compliance/autoexpanding-archiving.md

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

jcreigt committed:
Update autoexpanding-archiving.md

Fixed Outlook Desktop to Outlook for Windows.
Removed reference to "Within this preview" since it is out to Monthly Enterprise Channel now.

cabailey commented:
#sign-off

cabailey committed:
Merge branch 'public' into patch-3
